apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -1323,15 +1323,6 @@ suite('thread_list_ui', function() {
     });
 
     suite('respects l10n lib readiness', function() {
```

Comment:
I'm not sure this suite is useful; I think it was checking whether we were waiting for "once" to render threads, but it was removed some patches ago...